include/mlio/csv_reader.h

```diff
@@ -59,6 +59,10 @@ struct MLIO_API csv_params final {
     /// A boolean value indicating whether the dataset has a header row
     /// only in the first data store.
     bool has_single_header = false;
+    /// A boolean value indiciating whehter duplicate columns should be
+    /// renamed. If true, duplicate columns 'X', ..., 'X' will be renamed to
```

Comment:
Please limit the comment lines to 72 characters. This line exceeds it.

Reply:
Fixed.